app/lib/repository_check.rb

```diff
@@ -0,0 +1,20 @@
+class RepositoryCheck
+  def self.init
+    REPOSITORY_STATUS_CHECKS << name
+  end
+
+  def self.load_checks
+    Dir[File.join("#{Rails.root}/app/lib/repository_checks", "*.rb")].each do |file|
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Please use `Rails.root.join('path/to')` instead.](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5444834)